src/main/java/com/google/sps/data/ComputeAPIClientInterface.java

```diff
@@ -0,0 +1,49 @@
+package com.google.sps.data;
+
+import com.google.api.services.compute.model.Operation;
+import java.io.IOException;
+import java.util.Optional;
+
+/** Interface for the ComputerAPIClient class. */
+public interface ComputeAPIClientInterface {
+  /** 
+   * Creates an environment, calls each operation and tracks to see if they 
+   * are executed.
+   * @param {String} operation - the name of the method that should be called.
+   * @param {String} instanceName - the name of the instance being operated on.
+   * @param {Optional<String>} password - the password needed to access this 
+   *    instance. Not all operations requires a password.
+   */
+  void helper(String operation, String instanceName, 
+    Optional<String> password);
```

Comment:
not really sure what this is lol. This is not a good name. Helper is far too generic. I have no idea what this is supposed to do.